### PR TITLE
Add test case

### PR DESCRIPTION
**Pull request by yancyribbens**
Since there are test cases for other functions here, add test case as follow up for https://github.com/rust-bitcoin/rust-bitcoin/pull/3604.  I forgot to request this during review.

### CODE REVIEW

**Review by apoelstra — DISMISSED** [review]
ACK 618a785aa2a66b2bfd5ba357dc55b3144f5b644e; successfully ran local tests

**Comment by rustaceanrob** [comment]
Thanks, cheers

File: units/src/fee_rate.rs [hunk 1]
```diff
@@ -328,6 +328,9 @@ mod tests {
     fn fee_rate_from_sat_per_vb_test() {
         let fee_rate = FeeRate::from_sat_per_vb(10).expect("expected feerate in sat/kwu");
         assert_eq!(FeeRate(2500), fee_rate);
+
+        let fee_rate = FeeRate::from_sat_per_kvb(10);
+        assert_eq!(FeeRate(2), fee_rate);
     }
```
Comment (tcharding) [inline units/src/fee_rate.rs:334]: I know its simple but this breaks the test name, can we put this in a separate test?

```rust
    fn fee_rate_from_sat_per_kvb() {
        let fee_rate = FeeRate::from_sat_per_kvb(10);
        assert_eq!(FeeRate(2), fee_rate);
    }
```

For bonus points put a patch up front that removes the `_test` suffix from the other one.

Comment (yancyribbens) [reply units/src/fee_rate.rs]: I was being lazy because it's such a trivial test case.  I've added it it's own test.

**Review by apoelstra — APPROVED** [review]
ACK e7063fa181544e2d5f9e9a5521ad2a80c2b0eafc; successfully ran local tests

**Review by tcharding — DISMISSED** [review]
ACK e7063fa181544e2d5f9e9a5521ad2a80c2b0eafc

**Comment by apoelstra** [comment]
Needs rebase.

**Review by apoelstra — APPROVED** [review]
ACK 422365582964cf405529de3a1cf63d766b0d0e2e; successfully ran local tests

**Review by tcharding — APPROVED** [review]
ACK 422365582964cf405529de3a1cf63d766b0d0e2e

**Pull request merged**